src/python/system/shell.py

```diff
@@ -368,12 +368,6 @@ def get_interpreter(file_to_execute, is_blackbox_fuzzer=False):
   except KeyError:
     return None
 
-  # TODO(mbarbella): Remove this when fuzzers have been migrated to Python 3.
-  if (is_blackbox_fuzzer and interpreter == sys.executable and
```

Comment:
Can you clean up use of is_blackbox_fuzzer argument as it is not needed.

Reply:
Good catch. Done!